src/app/components/Slideshow/sliderComponents/Slides.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const Slide = props => {
+  const { index, activeIndex, slide } = props;
+
+  return (
+    <li className={index === activeIndex ? 'slider__slide slider__slide--active' : 'slider__slide'}>
+      <a href="https://megasport.ua/">
+        <img src={slide.content} alt="slideImg" />
```

Comment:
`alt` is hardcoded

Reply:
should it be got dynamically from server?